openiddict/Directory.Packages.props

```diff
@@ -0,0 +1,53 @@
+<Project>
+    <PropertyGroup>
+        <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    </PropertyGroup>
+
+    <!-- Package Versions for .NET 6.0 -->
+    <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '6.0'))">
+        <PackageVersion Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.1" />
+        <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0" />
+
+        <PackageVersion Include="OpenIddict" Version="5.2.0" />
```

Comment:
I have upgraded all the OpenIddict versions from 5.0.1 but have made no review of what has changed in those - might be best to revert and then upgrade them separately in like Audacia.Auth 4.1.0

Reply:
I think this upgrade is fine. On the face of it versions 5.1.0 and 5.2.0 of OpenIddict don't contain any changes that massively affect us.